dom.bs

```diff
@@ -990,34 +1004,43 @@ occurred. Each {{EventTarget}} has an associated list of
 <p>An <dfn export id=concept-event-listener>event listener</dfn> can be used to observe a specific
 <a>event</a>.
 
-<p>An <a>event listener</a> consists of a <b>type</b>, <b>callback</b>, and <b>capture</b>. An
+<p>An <a>event listener</a> consists of a <b>type</b>, <b>callback</b>, <b>capture</b> and <b>passive</b>. An
 <a>event listener</a> also has an associated <b>removed flag</b>, which is initially unset.
 
 <p class="note no-backref">The callback is named {{EventListener}} for historical reasons. As can be
 seen from the definition above, an <a>event listener</a> is a more broad concept.
 
 <dl class=domintro>
- <dt><code><var>target</var> . <a method lt="addEventListener()">addEventListener</a>(<var>type</var>, <var>callback</var> [, <var>capture</var> = false])</code>
+ <dt><code><var>target</var> . <a method lt="addEventListener()">addEventListener</a>(<var>type</var>, <var>callback</var> [, <var>options</var>])</code>
  <dd>
   Appends an <a>event listener</a> for <a>events</a> whose {{Event/type}} attribute value
   is <var>type</var>. The <var>callback</var> argument sets the <b>callback</b> that will
-  be invoked when the <a>event</a> is <a>dispatched</a>. When set to true,
-  the <var>capture</var> argument prevents <b>callback</b> from being invoked when
+  be invoked when the <a>event</a> is <a>dispatched</a>.
+
+  The <var>options</var> argument sets listener-specific options.  For compatibility this can be
+  just a boolean, in which case it determines the value of the <b>capture</b> option.
+
+  When set to true,
+  the <var>capture</var> option prevents <b>callback</b> from being invoked when
```

Comment:
How about `the capture member of <var>options</var>`?  Or do you prefer simply `the capture option`?